pkg/test/e2e/ccm-migration/ccm_migration_test.go

```diff
@@ -42,7 +42,7 @@ import (
 
 const (
 	userClusterPollInterval = 5 * time.Second
-	customTestTimeout       = 10 * time.Minute
+	customTestTimeout       = 30 * time.Minute
```

Comment:
Tried with 15 minutes, this is what we currently have in master, but it still timeouts although the migration worked fine.